src/IssueViz/Editor/LocationTagging/LocationTagger.cs

```diff
@@ -132,9 +132,16 @@ private void TranslateTagSpans(ITextSnapshot newSnapshot)
             foreach (var old in TagSpans)
             {
                 var newSpan = old.Span.TranslateTo(newSnapshot, SpanTrackingMode.EdgeExclusive);
-                old.Tag.Location.Span = newSpan; // update the span stored in the location visualization 
 
-                translatedTagSpans.Add(new TagSpan<IIssueLocationTag>(newSpan, old.Tag));
+                if (newSpan.Length != old.Span.Length)
+                {
+                    old.Tag.Location.InvalidateSpan();
```

Comment:
It wouldn't hurt to explain our decision here either e.g. `If the user has typed inside the tagged region we'll stop showing a squiggly for that item`